assignments/objects.js

```diff
@@ -49,16 +85,37 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  'name': 'Susan',
+  'age': 70,
+  sayName: function() {
+    return this['name'];
+  },
+  'child': {
+    'name': 'George',
+    'age': 50,
+    sayName: function() {
+      return this['name'];
+    },
+    'child': {
+      'name': 'Sam',
+      'age': 30,
+      sayName: function() {
+        return this['name'];
+      },
+    }
+  }
+}
 
 // Log the parent object's name
-
+console.log(parent['name']);
 // Log the child's age
-
+console.log(parent['child']['age']);
 // Log the name and age of the grandchild
-
+console.log(parent['child']['child']['name'],parent['child']['child']['age']);
```

Comment:
whew this one is crazy! it's times like these where dot notation seems easier to read